index.html

```diff
@@ -1,12 +1,476 @@
+                    </div>
+                    <div class="flex">
+                        <div class="column">
+                            <div class="column-img">
+                                <img class="image" src="img/illustration-lack-of-meaningful-wor.svg" />
+                            </div>
+                            <div class="column-subtitle">
+                                <h3>Untapped Opportunities</h3>
+                            </div>
+                            <div class="column-desc">
+                                <p>Among employees who leave a company, more than half do so because
+                                    of lack of job fit or mobility. The worst part? It’s not due to the lack of opportunities
+                                    within the company, but rather proactively creating what they want.</p>
+                            </div>
+                        </div>
+                        <div class="column">
+                            <div class="column-img">
+                                <img class="image" src="img/illustration-lack-of-direction.svg" />
+                            </div>
+                            <div class="column-subtitle">
+                                <h3>Lack of Action</h3>
+                            </div>
+                            <div class="column-desc">
+                                <p>Charting a career path can feel challenging at best. The process
+                                can feel hard and complex, causing employees to stop before they even start. Comfort zones
+                                are hard to break without the right guidance and support.</p>
+                            </div>
+                        </div>
+                        <div class="column">
+                            <div class="column-img">
+                                <img class="image" src="img/illustration-lack-of-support.svg" />
+                            </div>
+                            <div class="column-subtitle">
+                                <h3>Inefficient Support</h3>
+                            </div>
+                            <div class="column-desc">
+                                <p>Even when employees are presented with the right opportunity,
+                            career mobility is halted when they don’t have the right managerial support. Worse, they
+                            don’t have the tools or right connections to support their own career growth.</p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        
+        <section class="one-column ">
+            <img class="lines" src="img/bg-lines.svg" />
+
+            <div class="container">
+                <div class="content">
+
+                    <div class="flex-column-align-center column">
```

Comment:
Helper class looks bad. Separate them and not place in 1 row. 
Follow this example. But you can create your own. 
```
.flex { display: flex; }
.fd-column { flex-direction: column; }
.ai-center { align-items: center; }
```